ethereum/jsonrpc/src/test/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/queries/BlockchainQueriesTest.java

```diff
@@ -326,17 +326,18 @@ public void logsShouldBeFlaggedAsRemovedWhenBlockIsNotInCanonicalChain() {
     final Block fork = gen.block(options);
     final List<TransactionReceipt> forkReceipts = gen.receipts(fork);
 
-    final List<Block> reorgedChain = new ArrayList<>(blocks.subList(0, forkBlock));
-    reorgedChain.add(fork);
-    final List<List<TransactionReceipt>> reorgedReceipts =
-        new ArrayList<>(blockReceipts.subList(0, forkBlock));
-    reorgedReceipts.add(forkReceipts);
+    // final List<Block> reorgedChain = new ArrayList<>(blocks.subList(0, forkBlock));
```

Comment:
Did you mean to leave this snippet commented?

Reply:
Placeholder to show it was being deleted and to check for side effects.  I'll delete it.